python/GafferArnoldUI/ArnoldShaderUI.py

```diff
@@ -418,11 +436,15 @@ def __plugMetadata( plug, name ) :
 		return True
 
 	node = plug.node()
+	relativeName = plug.relativeName( node )
 	if isinstance( node, GafferArnold.ArnoldShader ) :
-		key = plug.node()["name"].getValue() + "." + plug.relativeName( node )
+		key = plug.node()["name"].getValue() + "." + relativeName
 	else :
 		# Other nodes hold an internal shader
-		key = plug.node()["__shader"]["name"].getValue() + "." + plug.relativeName( node )
+		key = plug.node()["__shader"]["name"].getValue() + "." + relativeName
+
+	if relativeName == "parameters" and name.startswith( "layout:activator:" ) :
+		return __metadata[key].get( name )( plug )
 
 	return __metadata[key].get( name )
```

Comment:
This had me scratching my head a bit before it clicked - I wonder if this way of expressing it might be a bit clearer, and a bit more open to extension in future?

```
result =  __metadata[key].get( name )
if callable( result ) :
    return result( plug )
else :
    return result
```

Reply:
Fixed in 868cd685db0dc60e0fca711e7512aa9434f69df5